scripts/build-types.mjs

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-console, no-process-exit, functional/no-expression-statement */
+import { exec } from './exec.mjs';
+
+/**
+ * Pass cli args to `turbo`.
+ *
+ * NOTE: don't try to pass arguments to `api-extractor` with `--`.
+ */
+const args = process.argv.slice(2);
+
+/**
+ * When in CI `api-extractor` shouldn't apply the local flag.
+ *
+ * `api-extractor` will error if an api change has occurred without being committed in an `api.md` file.
+ */
+const apiExtractorArgs = process.env.CI ? [] : ['--', '--local'];
+
+exec('./node_modules/.bin/turbo', [
+  'build:types',
+  '--filter=./packages/*',
+  ...args,
+  ...apiExtractorArgs,
+])
+  .then(() => process.exit(0))
+  .catch((err) => {
+    console.error(err);
+    process.exit(1);
+  });
```

Comment:
This is similar to how remix does their build scripts. It allows for more fine tuned control.